CONTRIBUTING.md

```diff
@@ -96,6 +96,64 @@ Closes #123
+
+Every pull request triggers a full run of testing in the CI system.
+The failures reported by the code style checker (aka the "linter") and the unit tests are usually
+clear and easy to fix (and can be avoided by running `npm t` locally before creating a commit).
+But when an integration, or e2e test, fails, it's sometimes useful to consult the log files
+for the run.
+
+Here's how to see more details for a failed run:
+
+1. The test results can be found by first going to the GitHub page for the PR (for an example, find an open PR at https://github.com/rancher-sandbox/rancher-desktop/pulls?q=is%3Aopen+is%3Apr),
+then look for the list of checks, find the `E2E/Integration Tests` entry, and click on the `Details` link for the first view.
+
+![Failing E2E tests link](./docs/assets/images/cilogs/screenshot01.png)
+
+This takes you to a more detailed view of the test, and which tests failed, but the log files are a few more clicks away.
+
+2. Scroll to the bottom and click on the link that reads `View more details on Cirrus CI`.
+
+![Link to Cirrus CI](./docs/assets/images/cilogs/screenshot02.png)
+3. Once the full test run has completed, the `Artifacts` button acts like a file-system browser.
+   The key here is to *not* click on the download button, but to click anywhere else on the row containing the button.
+
+![artifacts expansion button](./docs/assets/images/cilogs/screenshot03.png)
+4. This opens up a file browser. Now click on the list-view icon (the left of the two icons at the right of the row).
+
+
+![list-view button](./docs/assets/images/cilogs/screenshot04.png)
+5. Find the `.tar` file for the failed test, and click on it to download it into your designated `downloads` directory.
+
+
+![tar-file link](./docs/assets/images/cilogs/screenshot05.png)
+
+6. Because each test's tar file contains the same log files, it's a good idea to extract them into a separate
+directory. For example, if you're working with PR 5188:
+
+```bash
+Downloads$ mkdir pr5188
+Downloads$ cd pr5188
+Downloads/pr5188$ unzip ../playwright.zip
+Downloads/pr5188$ cd e2e/reports
+Downloads/pr5188/e2e/reports$ mkdir backend
+Downloads/pr5188/e2e/reports$ cd backend
+Downloads/pr5188/e2e/reports/backend$ tar xf ../backend.e2e.spec.ts-logs.tar
+Downloads/pr5188/e2e/reports/backend$ ls
+background.log     k8s.log              lima.serial.log    settings.log
+images.log         lima.ha.stderr.log   mock.log           steve.log
+integrations.log   lima.ha.stdout.log   networking.log     update.log
+k3s.log            lima.log             server.log         wsl.log
+```
+
+At this point, ideally,
+the failure in the integration run will be tied to an exception in a log file related to the change in the PR.
+
+Also note that due to code transpilation,
+the file names and line numbers in the log files will not usually correspond to the lines in the source code.
+You might want to add some `console.log` statements around the problematic area to highlight a problematic area.
```

Comment:
Repetitive phrasing here — it's really a problematic area ;)

Reply:
Well, they are *different* kind of problematic areas :)